### PR TITLE
Feature/kaldi speech recognizer

### PR DESCRIPTION
**Pull request by robertocaiwu**
- add support for Kaldi in mdr_speech_recognition script

### CODE REVIEW

File: mdr_speech/mdr_speech_recognition/ros/src/mdr_speech_recognition/speech_recognizer.py [hunk 1]
```diff
@@ -10,7 +11,15 @@ class SpeechRecognizer(object):
     def __init__(self):
         rospy.init_node("speech_recognizer")
         self.pub = rospy.Publisher("speech_recognizer", String, latch=True, queue_size=1)
+        self.model_directory = '{0}/speech/py-kaldi-asr/data/models/nnet3_en_f' \
```
Comment (robertocaiwu) [inline mdr_speech/mdr_speech_recognition/ros/src/mdr_speech_recognition/speech_recognizer.py]: This is the current directory of the Kaldi model, should I create a common folder and move it there?

**Comment by argenos** [comment]
In general it’s not a good idea to hard-code the paths, maybe check out how @minhnh does this for the perception models.

**Comment by PatrickNa** [comment]
I agree and we already discussed this a little during our meeting. Another question, which I am also not sure about is the following: Where do we store our models? When it comes to speech we will probably have a few different models. Then there are the perception models and so on. Do we have a place defined where those are going to be stored? @argenos

**Comment by robertocaiwu** [comment]
This request was closed in favor of merging #127.

**Pull request closed without merging**